modules/sdk-core/src/bitgo/staking/stakingWallet.ts

```diff
@@ -0,0 +1,198 @@
+/**
+ * @prettier
+ */
+import {
+  IStakingWallet,
+  StakingRequest,
+  TransactionsReadyToSign,
+  StakingTransaction,
+  StakingPrebuildTransactionResult,
+  StakingSignedTransaction,
+  StakingSignOptions,
+} from './iStakingWallet';
+import { BitGoBase } from '../bitgoBase';
+import { IWallet } from '../wallet';
+import { ITssUtils, RequestTracer, TssUtils } from '../utils';
+
+export class StakingWallet implements IStakingWallet {
+  private readonly bitgo: BitGoBase;
+  private readonly enterpriseId: string;
+
+  public wallet: IWallet;
+  public tssUtil: ITssUtils;
+
+  constructor(enterpriseId: string, wallet: IWallet, bitgo: BitGoBase) {
+    this.enterpriseId = enterpriseId;
```

Comment:
`enterpriseId` does not seem to be used, can we remove this?

Reply:
removed